cloud-controller-manager/osc/utils.go

```diff
@@ -370,11 +386,11 @@ func isEqualStringPointer(l, r *string) bool {
 	return *l == *r
 }
 
-func ipPermissionExists(newPermission, existing *ec2.IpPermission, compareGroupUserIDs bool) bool {
-	if !isEqualIntPointer(newPermission.FromPort, existing.FromPort) {
+func ipPermissionAWSExists(newPermission, existing *ec2.IpPermission, compareGroupUserIDs bool) bool {
```

Comment:
Osc maybye ?

Reply:
Not yet, like hasClusterAWSTag, it is still used, this is the process of the migration